conda_build.recipe/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+
+# Set the version info based on python version.
+VERSION="$(python setup.py --version)"
+
+# Add the git hash if it is missing.
+if [[ "${VERSION}" == *+ ]];
+then
+    VERSION="${VERSION}$(git rev-parse --short HEAD)"
```

Comment:
Not sure why it's missing here, but a version string can never end in ``+`` (in case git didn't work for a dev version, it'll be ``x.y.x.dev0+Unknown``)